src/Query/Operation/GetAll.php

```diff
@@ -0,0 +1,59 @@
+<?php
+declare(strict_types = 1);
+
+namespace TBolier\RethinkQL\Query\Operation;
+
+use TBolier\RethinkQL\Message\MessageInterface;
+use TBolier\RethinkQL\Query\QueryInterface;
+use TBolier\RethinkQL\RethinkInterface;
+use TBolier\RethinkQL\Types\Term\TermType;
+
+class GetAll extends AbstractOperation
+{
+    /**
+     * @var array
+     */
+    private $keys;
+
+    /**
+     * @var QueryInterface
+     */
+    private $query;
+
+    /**
+     * @param RethinkInterface $rethink
+     * @param MessageInterface $message
+     * @param QueryInterface $query
+     * @param array $keys
+     */
+    public function __construct(
+        RethinkInterface $rethink,
+        MessageInterface $message,
+        QueryInterface $query,
+        array $keys
+    ) {
+        parent::__construct($rethink, $message);
+
+        $this->query = $query;
+        $this->keys = $keys;
+        $this->rethink = $rethink;
+        $this->message = $message;
+    }
+
+    /**
+     * @inheritdoc
+     */
+    public function toArray(): array
+    {
+        return [
+            TermType::GET_ALL,
+            [
+                $this->query->toArray(),
+                [
+                    TermType::MAKE_ARRAY,
+                    $this->keys,
```

Comment:
This will not work for multiple values. We need to remove make_array and send each value as a new parameter.